includes/db-updates.php

```diff
@@ -1,5 +1,22 @@
 <?php
 /**
+ * Version 3.0.8
+ *
+ * Force update for older versions to call \dbdelta in install() method to fix column widths.
+ *
+ * @param string $db_version
+ * @param string $current_version
+ *
+ * @return string
+ */
+function wp_stream_update_308( $db_version, $current_version ) {
```

Comment:
Not sure if you'll need to update this or not: https://github.com/xwp/stream/blob/develop/classes/class-install.php#L337-L340